src/main/java/net/goldiriath/plugin/game/inventory/InventoryUtil.java

```diff
@@ -83,4 +117,52 @@ public static boolean storeInInventory(PlayerInventory inv, ItemStack stack) {
 
     }
 
+    public static WeaponType getWeaponType(Player player) {
```

Comment:
Perhaps, (semantics), a method like `getWeaponType(ItemStack stack)` is more desirable, since it is less strict on input parameters.